### ISSUE TITLE
scrolling necessary

### ISSUE THREAD

**Issue by HexeBianca** [danielramosg/sphereofearth#18]
On the 'What is a map' tab I need to scroll horizontally, to be able to read the full text.

**Comment by danielramosg** [comment]
In fullscreen? horizontally? Mmm, this must be because the font type and size are system dependent. It will be probably solved together with https://github.com/danielramosg/sphereofearth/issues/9.

**Comment by HexeBianca** [comment]
Yes, fullscreen and horizontally. There is plenty of (white) space on my two-screen desktop, the assigned space for the text is just too narrow.

**Comment by HexeBianca** [comment]
here is a screenshot with the new version without fullscreen:
![1](https://cloud.githubusercontent.com/assets/10153723/9665172/ced4e466-526e-11e5-883c-7d902d4450ac.jpg)

**Comment by danielramosg** [comment]
I think I fixed it, but test it in your screen. If it looks ok, you can close the issue.

**Comment by HexeBianca** [comment]
Now, everything is visible, but there is still a horizontal scrollbar, which does not disappear, no matter how much I increase the width of the window:
![1](https://cloud.githubusercontent.com/assets/10153723/9715838/9afaa29a-5564-11e5-9d7b-1f3091bd5b34.jpg)
